src/re_complete/app.cljs

```diff
@@ -146,14 +146,18 @@
                        (get-in db [:re-complete :linked-components linked-component-key :change-index])
                        (get-in db [:re-complete :linked-components linked-component-key :options :trim-chars])
                        %
-                       selected-word)) 
+                       selected-word))
           (clear-complete-items linked-component-key)
           (clear-selected-item linked-component-key))))
 
 (defn keys-handling [linked-component-key onclick-callback]
-  (events/listen js/window "keydown"
-                 (fn [e]
-                   (let [key-code (.-keyCode e)]
-                     (when (#{13 38 40 9} key-code)
-                       (.preventDefault e)
-                       (dispatch [:keys-handling linked-component-key key-code onclick-callback]))))))
+  (.addEventListener js/window "keydown"
```

Comment:
Genuine question, shouldn't `events/listen` do the same as `.addEventListener`, why choosing one over the other?